internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -289,3 +314,54 @@ func setCPUQuota(cpuMountPoint, parentDir string, c *oci.Container, enable bool)
 
 	return nil
 }
+
+func setRPSLoadBalancing(c *oci.Container, ncpus int, sysDevicesDir string, enable bool) error {
+	lspec := c.Spec().Linux
+	if lspec == nil ||
+		lspec.Resources == nil ||
+		lspec.Resources.CPU == nil ||
+		lspec.Resources.CPU.Cpus == "" {
+		return errors.Errorf("find container %s CPUs", c.ID())
+	}
+
+	machineCPUs, err := cpuset.Parse(fmt.Sprintf("0-%d", ncpus-1))
+	if err != nil {
+		return err
+	}
+
+	machineCPUsMask := cpuNumToCPUMask(ncpus)
```

Comment:
Why a new method? Can't you get the mask form machineCPUs? It represents the same data, right?

Reply:
We actually don't have access to this info, but indeed reading  /sys/devices/system/cpu/online will give us the info as a set already, and also we will not need to use the nproc tool. I'll make the changes, thanks.